src/firebolt/async_db/_types.py

```diff
@@ -263,3 +258,23 @@ def process_token(token: Token) -> Token:
         )
 
     return formatted_sql
+
+
+def format_sql(query: str, parameters: Sequence[ParameterType]) -> str:
+    return format_statement(parse_sql(query)[0], parameters)
+
+
+def split_format_sql(
+    query: str, parameters: Sequence[Sequence[ParameterType]]
```

Comment:
Could you add a short docstring for this function? Despite not being public it would be useful to have down the line.

Reply:
Yes, sure